src/components/Editor/index.js

```diff
@@ -525,10 +525,12 @@ class Editor extends PureComponent {
     const { sourceId } = selectedLocation;
 
     if (bp) {
+      // NOTE: it's possible the breakpoint has slid to a column
+      column = column || bp.location.column;
```

Comment:
I am wondering which one should take precedence here. If both `column` and `bp.location.column` are defined, which one would be more important? From the comment it sounds like `bp.location.column` -- because it should be overwriting to account for sliding? But I'm not too sure, sliding bps are hard!